services/ui-src/src/components/modals/CloseEntityModal.tsx

```diff
@@ -0,0 +1,46 @@
+import React from "react";
+// components
+import { Text } from "@chakra-ui/react";
+import { Modal } from "components";
+
+// types
+import { AnyObject, EntityShape } from "types";
+import { renderHtml } from "utils";
+
+export const CloseEntityModal = ({
+  selectedEntity,
+  verbiage,
+  modalDisclosure,
+}: Props) => {
+  const closing = false;
+  const closeProgramHandler = async () => {
+    // eslint-disable-next-line no-console
+    console.log("Entity closed", selectedEntity);
+    modalDisclosure.onClose();
+  };
+
+  return (
+    <Modal
+      onConfirmHandler={closeProgramHandler}
+      modalDisclosure={modalDisclosure}
+      submitting={closing}
+      content={{
+        heading: verbiage.closeOutModal.closeOutModalTitle,
+        subheading: verbiage.closeOutModal.closeOutModalBodyText,
+        actionButtonText: verbiage.closeOutModal.closeOutModalConfirmButtonText,
```

Comment:
Nit: Since these are nested variables we're pulling to fill out the heading/subheading/actionbuttontext we could create a const and call it from there. so like 
`const closeOutVerbiage = verbiage.closeOutModal` and then you can call `closeOutVerbiage.closeOutModalTitle`